spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/convert/DefaultDatastoreEntityConverterTests.java

```diff
@@ -301,45 +303,66 @@ public void testUnsupportedTypeWrite() {
 
 	@Test
 	public void testCollectionFieldsUnsupportedCollection() {
+
 		this.thrown.expect(DatastoreDataException.class);
 		this.thrown.expectMessage("Unable to read " +
 				"org.springframework.cloud.gcp.data.datastore.core.convert.TestDatastoreItemCollections entity;");
-		this.thrown.expectMessage("Unable to read property doubleSet;");
+		this.thrown.expectMessage("Unable to read property beanContext;");
+
 		this.thrown.expectMessage(
 				"Failed to convert from type [java.util.ArrayList<?>] " +
-						"to type [com.google.common.collect.ImmutableSet<?>]");
+						"to type [org.springframework.cloud.gcp.data.datastore.core.convert.TestDatastoreItemCollections$ComparableBeanContextSupport]");
+
+		ComparableBeanContextSupport ComparableBeanContextSupport = new ComparableBeanContextSupport();
```

Comment:
Typical Java code style to have classes capitalized, but variables in camel case starting with non-capitalized letter.